src/refmvs.rs

```diff
@@ -54,6 +56,7 @@ impl From<[i8; 2]> for RefMvsRefPair {
 pub struct RefMvsMvPair {
     pub mv: [Mv; 2],
 }
+const _: () = assert!(mem::size_of::<RefMvsTemporalBlock>() == 5);
```

Comment:
I think this one was accidentally copied. Should probably be the missing `RefMvsMvPair` check.